packages/lib/checkRateLimitAndThrowError.ts

```diff
@@ -48,6 +47,7 @@ export async function checkSMSRateLimit({
 }
 
 async function changeSMSLockState(identifier: string, status: SMSLockState) {
+  const prisma = (await import("@calcom/prisma")).prisma;
```

Comment:
Is there a reason for this?

Reply:
This was one of the first tests I fixed, will fix in a different way